ktor-server/ktor-server-plugins/ktor-server-call-logging/jvm/src/io/ktor/server/plugins/callloging/MDCHook.kt

```diff
@@ -20,3 +21,12 @@ internal fun MDCHook(phase: PipelinePhase) = object : Hook<suspend (ApplicationC
         }
     }
 }
+
+internal object ResponseSent : Hook<suspend (ApplicationCall) -> Unit> {
```

Comment:
Why we can't add `suspend` to the existing `ResponseSent` [hook](https://github.com/ktorio/ktor/blob/c333fb95143cbb9dc94857e42295bc0d0aeaea23/ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/application/hooks/CommonHooks.kt#L108)? Can it break something??